mockkubeapiserver/storage/interface.go

```diff
@@ -0,0 +1,33 @@
+package storage
```

Comment:
I like this style in storage, and the hook (I misread the rationale of Hook until seeing the "CRDHook")